app/controllers/api/v1/application_controller.rb

```diff
@@ -8,6 +8,10 @@ def authenticate
           ActiveSupport::SecurityUtils.secure_compare(token, Rails.application.config.authentication_token)
         end
       end
+
+    private
+
+      def check_terms_accepted; end
```

Comment:
Do we need this on both controllers? Can this not just be on the v2 controller, along with the before action?